routing/hostname_pattern.rst

```diff
@@ -193,7 +193,8 @@ instance, if you want to match both ``m.example.com`` and
             <?xml version="1.0" encoding="UTF-8" ?>
             <routes xmlns="http://symfony.com/schema/routing"
                 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-                xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
+                xsi:schemaLocation="http://symfony.com/schema/routing
```

Comment:
I think @atailouloute is right. We have literally thousands of these XML URLs and none of them use HTTPS. We recently added HTTPS for every link that points to symfony.com (see #7481) but we left on purpose these links unchanged. The reason is that it could generate lots of conflicts when merging other PRs ... but maybe I'm wrong and changing to HTTPS would create no conflicts. In any case, I think we should decide to do that (or not) in a separate PR. Thanks!